pkg/logs/internal/launchers/journald/launcher.go

```diff
@@ -51,7 +55,7 @@ func (l *Launcher) run() {
 	for {
 		select {
 		case source := <-l.sources:
-			identifier := source.Config.Path
+			identifier := tailer.Identifier(source.Config)
```

Comment:
Would it make sense to add a warning here? Users are now more likely to have multiple configs, a warning may be helpful if config_id is left out by mistake.